multidim-interop/dockerBuildWrapper.sh

```diff
@@ -9,6 +9,8 @@ if [[ -n "${CI}" ]] && [[ -n "${AWS_BUCKET}" ]]; then
     --cache-from type=s3,mode=max,bucket=$AWS_BUCKET,region=$AWS_REGION,prefix=buildCache,name=$IMAGE_NAME \
     "$@"
 else
-	docker build -t $IMAGE_NAME "$@"
+    docker buildx build \
+    --load \
+	-t $IMAGE_NAME "$@"
```

Comment:
Would it make sense to do something like:


```suggestion
    docker buildx build \
    --load \
    $CACHING_OPTIONS \
	-t $IMAGE_NAME "$@"
```

Where `$CACHING_OPTIONS` is conditionally defined instead of the entire command?